16/alpine3.11/Dockerfile

```diff
@@ -0,0 +1,100 @@
+FROM alpine:3.11
+
+ENV NODE_VERSION 16.0.0
+
+RUN addgroup -g 1000 node \
+    && adduser -u 1000 -G node -s /bin/sh -D node \
+    && apk add --no-cache \
+        libstdc++ \
+    && apk add --no-cache --virtual .build-deps \
+        curl \
+    && ARCH= && alpineArch="$(apk --print-arch)" \
+      && case "${alpineArch##*-}" in \
+        x86_64) \
+          ARCH='x64' \
+          CHECKSUM="" \
```

Comment:
We're blocked till the Alpine binaries are built https://unofficial-builds.nodejs.org/download/release/v16.0.0/

Reply:
I see, only skimmed over the Dockerfile. I thought I had it building locally. out of curiosity, what is the "Building from source" condition for?

Reply:
I think mostly a fail safe for anyone doing a manual build. The dockerhub image layers are built using the binary version though, so we wait till they're ready

Reply:
Also for the other architectures beyond the three that get built in the unofficial builds project. 😅

Reply:
Yeah, I was going to caveat that around the s360 etc.., but it's a confusing story